addons/web/static/src/model/record.js

```diff
@@ -167,17 +167,21 @@ export class Record extends Component {
         "onWillSaveRecord?",
     ];
     setup() {
-        if (this.props.fields) {
-            this.fields = this.props.fields;
+        const { activeFields, fieldNames, fields, resModel } = this.props;
+        if (!activeFields && !fieldNames) {
+            throw Error(`Record props should have either a "resModel" key or a "fieldNames" key`);
```

Comment:
Hello :wave:
I think there is a typo in `resModel`, should it be `activeFields`?
I noticed it while debugging the error during a task. ^^